### DISCUSSION TITLE
An issue with non stand-alone mods

### DISCUSSION THREAD

**Discussion by azzamsa** [manna-harbour/miryoku#92] [Q&A] [answered]
Hi, this is my first time using Miryoku. After using my own layout for over a year. I use Gnome for my DE, and I need a single tap of modifier (`super`) to show all the windows. 

Currently, using vanilla Miryoku, I can hold `a` for more than 2 second to show all windows. but, 1) it takes long time, and 2) the second press resulting in "a" not `super`. So, instead of hiding all windows, it search for apps.

![image](https://user-images.githubusercontent.com/17734314/172329154-9f4c0f5a-663b-4dea-9f42-9e217c4fcbbf.png)

My previous layout features a stand-alone mods, so I can just do tap->tap to show->hide all windows.
Any ideas?

Thanks for Miryoku.

**Comment by manna-harbour** [comment, marked as answer]
The mods on layers are basic mods, so you can tap those. If you have extra keys you could also map one of those for convenience, see https://github.com/manna-harbour/miryoku/discussions/85.

**Reply by azzamsa** [reply]
>  If you have extra keys you could also map one of those for convenience,

I would like to try vanilla Miryoku to the fullest extent possible :)

> The mods on layers are basic mods, so you can tap those.

I think I can't "tap" those keys, tapping will trigger the alphas, not the mods. So, I need to "hold" them for a while (2-3 seconds).
To be able to do "tap", I need to activate the nav layer using my left thumb and press the a using my pinky. This way, I can activate `super` using "tap". tap-tap, it works. But it leads me to press more than one key with one hand, which will lead to RSI at some point.

Other than extra keys, do you have other ideas?

**Reply by azzamsa** [reply]
Solved. I use two keys to `show overview`. `Super-S`, which also default in Gnome.

**Reply by manna-harbour** [reply]
Yes, I meant hold Nav then tap the mod. For normal use of mods and layer change you can hold all simultaneously, and the fingers stay on home, in order to not stress the hand. But in order to tap the mod you have to hold the layer first and then tap the finger, which is still on home, but that movement might be more stressful as you noted. In that case your alternative binding would be better.
